readme.md

```diff
@@ -35,7 +35,7 @@ answer goes here
 Describe the difference between a relative and absolute path, give an example of each.
 
 ```
-answer goes here
+Relative paths are links to sources relative to the local directory we are working at, while absolute paths are links to pages and content that are somewhere in the internet. For instance, if there is a file called Photo1.jpg that I want to link, a relative path would be /images/photo1.jpg. An absolute path would be a whole website address: http://www.website.com/images/photo1.jpg.
 
```

Comment:
You got it with relative paths, but really absolute paths refers to writing out the whole path for a file starting at the root. The root level on a mac would be user. so it'd be like user/gio/desktop/wdi/portfolio/index.html